### ISSUE TITLE
Function has a different Type !

### ISSUE THREAD

**Issue by Xunil1337** [Impact123/CallAdmin#11]
/opt/SteamCompile/addons/sourcemod/scripting/include/calladmin.inc(165) : error 181: function argument named 'type' differs from prototype

```c++
/**
 * Called when the serverdata data is changed.
 *
 * @param convar       Handle to the convar which was changed.
 * @param type         Type of data which was changed.
 * @param oldVal       Value of data before change.
 * @param newVal       Value of data after change.
 * @noreturn
 */
forward void CallAdmin_OnServerDataChanged(ConVar convar, ServerData type, const char[] oldVal, const char[] newVal);

enum ServerData
{
	ServerData_HostName,     /**< This is the hostname of the server, gathered from the `hostname` convar */
	ServerData_HostIP,       /**< This is the hostip of the server, gathered and converted from the `hostip` convar */
	ServerData_HostPort      /**< This is the hostport of the server, gathered from the `hostport` convar */
};
```

https://github.com/Impact123/CallAdmin/blob/development/gameserver/calladmin.inc#L175

**Comment by Impact123** [comment]
What exactly did you try to do? And what version of the compiler did you use?

**Comment by Xunil1337** [comment]
I only recompile it not more !

**Comment by Impact123** [comment]
I can't help you if you don't give me more information. It compiles fine when following [the guide](https://github.com/Impact123/CallAdmin/wiki/Compiling).

**Comment by Xunil1337** [comment]
Bad Support !

**Comment by popoklopsi** [comment]
I rather think bad scripter ;)

**Comment by Xunil1337** [comment]
I am not the scripter, i only change the domain from example to my domain and compile it new no compile errors ... but okay ...

**Comment by Impact123** [comment]
You don't have to compile to change the domain.

**Comment by Rainyan** [comment]
I can confirm being unable to compile this both on SM 1.7 stable and the latest snapshot of SM 1.8.
Attempts to compile result in the error:
"error 181: function argument named 'type' differs from prototype"

I have all the dependencies mentioned in https://github.com/Impact123/CallAdmin/wiki/Compiling in place. I'm trying to compile on Windows 8, 64bit, if that makes a difference.

**Comment by Impact123** [comment]
As already said to Xuinil, i need steps to reproduce in order to help. We let this build automatically via travis and our our own build server and it works fine, you can view the build steps for [here](https://travis-ci.org/Impact123/CallAdmin/). Make sure that All your includes are up2date too. Maybe [this](https://forums.alliedmods.net/showpost.php?p=2312340&postcount=1217) helps?

**Comment by Xunil1337** [comment]
There is your Code : https://forums.alliedmods.net/showthread.php?t=213670

**Comment by popoklopsi** [comment]
It's better to use the development files from github

**Comment by Impact123** [comment]
As said in [the link i posted](https://forums.alliedmods.net/showpost.php?p=2312340&postcount=1217), that code was released before 1.7 which broke compatibility with a lot of plugins, including CallAdmin.
